routers/repo/pull.go

```diff
@@ -1066,3 +1069,73 @@ func DownloadPullPatch(ctx *context.Context) {
 		return
 	}
 }
+
+// UpdatePullRequestTarget change pull request's target branch
+func UpdatePullRequestTarget(ctx *context.Context) {
+	issue := GetActionIssue(ctx)
+	pr := issue.PullRequest
+	if !issue.IsPull {
+		return
```

Comment:
ctx Writen should be moved before this if and here should return not found probably